modules/travel_pay/app/services/travel_pay/claims_client.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'securerandom'
+require_relative './base_client'
+
+module TravelPay
+  class ClaimsClient < TravelPay::BaseClient
+    ##
+    # HTTP GET call to the BTSSS 'claims' endpoint
+    # API responds with travel pay claims including status
+    #
+    # @return [TravelPay::Claim]
+    #
+    def get_claims(veis_token, btsss_token)
+      request_claims(veis_token, btsss_token)
```

Comment:
This seems redundant. @kjduensing are you opposed to moving the `request_claims` code into this function block now that there's not the whole "go get the tokens" required before it can be called?

Reply:
Good catch. Not opposed at all. Very much pposed.